### PR TITLE
add precourse

### PR DESCRIPTION
**Pull request by bcasasus**

### CODE REVIEW

File: staff/bernat-casasus/precourse/airline/project-airline.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+function lastFiveFlights(object){
+		var objLength = object.length-5;
+		var arrayDestinations = [];
+		object.forEach(function(obj){
+
+		if(obj.id === objLength){
+			objLength++;
+			arrayDestinations.push(obj.to);
+			}
+	})
```
Comment (Javier1177) [inline staff/bernat-casasus/precourse/airline/project-airline.js:92]: Falta semicolon, bro.

File: staff/bernat-casasus/precourse/airline/project-airline.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+		
+}
+
+function flightsWithSacle(object){
+	var count = 0;
+	object.forEach(function(obj){
+		if(obj.scale === true){
+			count++;
+			}
+	})
```
Comment (Javier1177) [inline staff/bernat-casasus/precourse/airline/project-airline.js:78]: You missed the fcking semicolon again, bro.

File: staff/bernat-casasus/cv/index.html [hunk 3]
```diff
@@ -7,53 +7,78 @@
...
-      </div>
-    </div>
+    <!-- Main -Divided by Secctions -->
+        <main>
+        <!-- 1 Section - Article - Curriculum Vitae -->
+            <section>
+                <h2>Curriculum Vitae</h2>
+                <article>
+                    <h5>Ragat Jonsson 12/12/1990</h5>
+                    <img src="./person.jpg" alt="Ragat Jonsson" width="200" height="250">
```
Comment (SergioCastilloSorribes) [inline staff/bernat-casasus/cv/index.html]: La imagen es mejor meterla dentro de la etiqueta <figure>.

**Review by SergioCastilloSorribes — CHANGES_REQUESTED** [review]

File: staff/bernat-casasus/portfolio/index.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    </head>
+    <body>
+            <header>
+                <h1>Jonsson Portfolio</h1>
+            </header>
+            <main>
+                <section>
+                    <h2>Green Alternative</h2>
+                    <figure>IMG</figure>
+                    <figurecaption>
```
Comment (alexgbor) [inline staff/bernat-casasus/portfolio/index.html]: It's better to put figurecaption inside the figure tag.

File: staff/bernat-casasus/portfolio/index.html [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+                                    <p>Sunt in culpa qui officia deserunt mollit anim id est Sunt in culpa qui off.</p>
+                                    <footer>
+                                        Third sentence in <cite><a href="http://www.george-orwell.org/1984/0.html"><i>Sunt Eighty-Four</i></a> by Jhon Mike (Part 7, Chapter 2)</cite>.
+                                    </footer>
+                                </blockquote>
+                        </article>
+                    </section>
+                    <hr>
+                <footer>
+                        <address>
```
Comment (alexgbor) [inline staff/bernat-casasus/portfolio/index.html:59]: A ul inside the address tag should be useful later on when adding a css file.

Comment (bcasasus) [reply staff/bernat-casasus/portfolio/index.html:59]: Updated!

File: staff/bernat-casasus/cv/index.html (hunk 3 above)
Comment (bcasasus) [reply staff/bernat-casasus/cv/index.html]: Done!

File: staff/bernat-casasus/portfolio/index.html (hunk 5 above)
Comment (alexgbor) [reply staff/bernat-casasus/portfolio/index.html:59]: Good job!

**Pull request merged**